apps/obsidian/src/components/TldrawView.tsx

```diff
@@ -135,9 +135,15 @@ export class TldrawView extends TextFileView {
 
   private createReactRoot(entryPoint: Element, store: TLStore) {
     const root = createRoot(entryPoint);
+    if (!this.file) return;
+
     root.render(
       <React.StrictMode>
-        <TldrawPreviewComponent store={store} isReadonly={false} />
+        <TldrawPreviewComponent
+          store={store}
```

Comment:
<img width="1181" height="289" alt="image" src="https://github.com/user-attachments/assets/6430970b-01da-4c9e-8b9c-8754543c6848" />

Do you see these lint warnings in your IDE?  If not, install the ESLint plugin.

Ideally we deal with these prior to merging, but in this case we can have a linting round before merging to main if they weren't showing up in your IDE.